api/src/test/java/org/openmrs/api/PersonServiceTest.java

```diff
@@ -363,6 +364,17 @@ public void parsePersonName_shouldParseTwoPersonNameWithoutComma() throws Except
 		assertEquals("Doe", pname2.getFamilyName());
 		assertEquals("John", pname2.getGivenName());
 	}
+
+	@Test
+	@Verifies(value = "should set sort weight to one when all person attribute types are empty", method = "savePersonAttributeType(PersonAttributeType)")
+	public void saverPersonAttributeType_shouldSetSortWeightToOneWhenAllPersonAttributeTypesAreEmpty() throws Exception {
+		PersonService service = Context.getPersonService();
```

Comment:
I am working on this ticket with a class mate and he was in charge of this test method. I have asked him to explain this method.